.github/workflows/experimental-wasm-release.yml

```diff
@@ -0,0 +1,48 @@
+name: WASM Experimental builds
+on:
+  push:
+    branches:
+      - experimental-wasm
+jobs:
+  build:
+    if: ${{ github.repository_owner == 'cloudflare' }}
+    name: Build and release `wrangler@wasm` to NPM
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Use Node.js 16.7
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.7
+          cache: "npm"
+
+      - uses: actions/cache@v2
+        id: node-modules-cache
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+
+      - name: Install NPM Dependencies
+        run: npm install
+
+      - name: Modify package.json version
+        run: node .github/version-script.js
+
+      - name: Build
+        run: npm run build
+        working-directory: packages/wrangler
+
+      - name: Check for errors
+        run: npm run check
```

Comment:
I would ditch this so that linting/formatting errors do not stop them from getting a distributable.

Reply:
I would push back on this. lint/format happens in IDE anyway, and besides this also does typechecking. I'd rather not have broken commits once we eventually do merge.